crates/cairo-coverage/src/input/sierra_to_cairo_map.rs

```diff
@@ -61,7 +61,10 @@ fn find_statement_origin(
         .zip(function_names)
         // TODO: We should probably filter by path to user project not by path to cache
         // TODO: Can get this from source_sierra_path in call trace
-        .find(|((file_location, _), _)| !file_location.contains("com.swmansion.scarb"))
+        .find(|((file_location, _), _)| {
+            !file_location.contains("com.swmansion.scarb")
+                && !file_location.contains(".cache/scarb")
```

Comment:
I guess this is not strictly related to this PR?

Reply:
This fix ensures that the tests pass on the CI. It will be redesigned in issue [#55](https://github.com/software-mansion/cairo-coverage/issues/55).